localstack/services/apigateway/provider.py

```diff
@@ -457,7 +457,27 @@ def get_method(
     ) -> Method:
         response: Method = call_moto(context)
         remove_empty_attributes_from_method(response)
-        remove_empty_attributes_from_integration(response.get("methodIntegration"))
+        method_integration = response.get("methodIntegration")
+        if method_integration:
```

Comment:
nit:
```suggestion
        if method_integration := response.get("methodIntegration")
```

Reply:
good catch, I was first aiming for 2 conditions and when I removed one I forgot the walrus :D Thanks!